packages/cli/src/platform/auth/login.ts

```diff
@@ -106,6 +106,7 @@ export class Login implements Command {
 
     {
       const writeResult = await credentialsFile.save({ token: callbackResult.token })
+      // @ts-ignore: Expected 0-1 arguments, but got 2.
```

Comment:
Why is this necessary? Do we have outdated typings or target older environment than we actually require? `Error` does take 0-2 arguments in modern JavaScript.

Reply:
Fixing this in a follow-up PR.